drivers/ethernet/eth_adin2111.c

```diff
@@ -27,6 +27,8 @@ LOG_MODULE_REGISTER(eth_adin2111, CONFIG_ETHERNET_LOG_LEVEL);
 
 #define DT_DRV_COMPAT adi_adin2111
 
+#define ADIN2111_ETH_MAC_ADDR_LEN		6U
```

Comment:
Maybe it is time to have a MAC address length in the `include/zephyr/net/ethernet.h`?
@jukkar, @rlubos - what do you think?

Reply:
Good point. I thought we already had a length defined there but I remembered wrong, please send a patch that defines one.

Reply:
thanks, i can do it too

Reply:
done